lib/resources/http.rb

```diff
@@ -22,23 +22,28 @@ class Http < Inspec.resource(1)
         its('Content-Length') { should cmp 258 }
         its('Content-Type') { should cmp 'text/html; charset=UTF-8' }
       end
-
-      # properly execute the HTTP call on the scanned machine instead of the
-      # machine executing InSpec. This will be the default behavior in InSpec 2.0.
-      describe http('http://localhost:8080', enable_remote_worker: true) do
-        its('body') { should cmp 'local web server on target machine' }
-      end
     "
 
     def initialize(url, opts = {})
       @url = url
       @opts = opts
 
-      if use_remote_worker?
-        return skip_resource 'curl is not available on the target machine' unless inspec.command('curl').exist?
-        @worker = Worker::Remote.new(inspec, http_method, url, opts)
-      else
+      # Prior to InSpec 2.0 the HTTP test had to be instructed to run on the
+      # remote target machine. This warning will be removed.
```

Comment:
This is a bit confusing... when will it be removed? 🙂 

How about we update this comment to "This warning will be removed after a few months to give users an opportunity to remove the unused option from their profiles."